apps/nextjs-website/react-components/components/PreFooter/PreFooter.tsx

```diff
@@ -69,27 +84,38 @@ const PreFooter = (props: PreFooterProps) => {
           background={!background ? backgroundColor : 'transparent'}
           sx={styles.container}
         >
-          <Box sx={styles.main(isSmallScreen)}>
+          <Box sx={styles.main(isSmallScreen, layout)}>
             <Typography
-              variant='h4'
+              variant='h6'
               color={theme === 'dark' ? 'white' : 'black'}
-              mb={isSmallScreen ? 2 : 'unset'}
+              mb={isSmallScreen || layout === 'center' ? 2 : 'unset'}
+              sx={{
+                fontSize: { xs: '18px', md: '24px' },
+                marginRight:
+                  layout === 'left' && !isSmallScreen ? '16px' : 'unset',
+              }}
             >
               {title}
             </Typography>
+
             {storeButtons?.hrefGoogle || storeButtons?.hrefApple ? (
```

Comment:
```suggestion
            {storeButtons ? (
```
Check storeButtons existence directly.